Chapter7/enum.erl

```diff
@@ -0,0 +1,37 @@
+-module(enum).
+-export([day/1, month/1]).
+
+% Exercise 7-8: Enumerated Types
+
+-define(DAY(Day),
+	case Day of
```

Comment:
This is not anyhow different from a function. I don't think there's a place for macro.